parquet-common/src/main/java/org/apache/parquet/hadoop/metadata/CompressionCodecName.java

```diff
@@ -30,7 +30,8 @@ public enum CompressionCodecName {
   LZO("com.hadoop.compression.lzo.LzoCodec", CompressionCodec.LZO, ".lzo"),
   BROTLI("org.apache.hadoop.io.compress.BrotliCodec", CompressionCodec.BROTLI, ".br"),
   LZ4("org.apache.hadoop.io.compress.Lz4Codec", CompressionCodec.LZ4, ".lz4"),
-  ZSTD("org.apache.parquet.hadoop.codec.ZstandardCodec", CompressionCodec.ZSTD, ".zstd");
+  ZSTD("org.apache.parquet.hadoop.codec.ZstandardCodec", CompressionCodec.ZSTD, ".zstd"),
+  LZ4_RAW("org.apache.parquet.hadoop.codec.Lz4RawCodec", CompressionCodec.LZ4_RAW, ".lz4");
```

Comment:
should there be a different extension?  I forget what we did in the C++ implemetation.

Reply:
We don't register file extensions in C++.
That said, ".lz4" probably can't be appropriate for the Hadoop LZ4 codec.

Reply:
Change the suffix to ".lz4raw"